webserver/webserver/src/main/java/io/helidon/webserver/http/ServerResponseBase.java

```diff
@@ -47,6 +51,20 @@
 @SuppressWarnings("unchecked")
 public abstract class ServerResponseBase<T extends ServerResponseBase<T>> implements RoutingResponse {
 
+    /**
+     * Stream status trailer name.
+     */
+    protected static final HeaderName STREAM_STATUS_NAME = HeaderNames.create("stream-status");
```

Comment:
`stream-status` is not a standard header as far as I know, it should start with `x`, please check if this is OK

Reply:
We have introduced this trailer in Helidon 2.x after mutual agreement, [here is the discussion](https://github.com/helidon-io/helidon/issues/2107). But we can open the issue again, rather before RC release.